distribution/lib/Standard/Base/0.0.0-dev/src/Data/Number/Extensions.enso

```diff
@@ -209,7 +209,8 @@ Integer.up_to n = Range this n
 
          1.equals 1.0000001 epsilon=0.001
 Number.equals : Number -> Number -> Boolean
-Number.equals that epsilon=0.0 = (this - that).abs <= epsilon
+Number.equals that epsilon=0.0 =
+    if this == that then True else (this - that).abs <= epsilon
```

Comment:
```suggestion
    (this == that) || ((this - that).abs <= epsilon)
```
?

Reply:
Right!

Somehow, I had the misconception that `(this - that).abs <= epsilon` will throw if `this = that = infinity` and since `||` is eager, it would be an issue. But no - the `-` just returns a `NaN` and `abs` forwards it, yielding `False` at the comparison. So you are right that this can be changed - will do